web/packages/teleterm/src/ui/services/workspacesService/documentsService/types.ts

```diff
@@ -329,7 +336,8 @@ export type Document =
   | DocumentConnectMyComputer
   | DocumentVnetDiagReport
   | DocumentVnetInfo
-  | DocumentAuthorizeWebSession;
+  | DocumentAuthorizeWebSession
+  | DocumentDesktopSession;
```

Comment:
Side note, I usually add new stuff as second to last so that the git diff shows only a single-line change. 😏